docs/api-routes/edge-api-routes.md

```diff
@@ -0,0 +1,70 @@
+---
+description: You can add the dynamic routes used for pages to API Routes too. Learn how it works here.
+---
+
+# Edge API Routes (Beta)
+
+<details open>
+  <summary><b>Examples</b></summary>
+  <ul>
+    <li><a href="https://github.com/vercel/examples/blob/main/edge-functions/api-route/api/edge.ts">Basic Edge API Route</a></li>
+  </ul>
+</details>
+
+API Routes can be configured to run at the Edge using an experimental flag. When deployed on Vercel, these API Routes run as [Edge Functions](https://vercel.com/docs/concepts/functions/vercel-edge-functions).
+
+Edge API Routes are similar to API Routes but with different infrastructure:
+
+- On every invocation API Routes are run from the same region
+- Edge API Routes are copied across the [Vercel Edge Network](https://vercel.com/docs/concepts/edge-network/overview), so on every invocation, the region that is closets to you will run the function, reducing latency massively
+
+Unlike API Routes, Edge API Routes:
+
+- Can stream responses
+- Run _after_ the cache
+- Can cache responses
+- Have zero cold starts
+
+## How to create an Edge API Route
+
+Edge API Routes have the same signature as [Edge Middleware](/docs/advanced-features/middleware), and support the same helpers from [`next/server`](/docs/api-reference/next/server). Note that the below examples use TypeScript, though this is **not** a requirement.
+
+### API Route
+
+```typescript
+import type { NextApiRequest, NextApiResponse } from 'next'
+
+export default (req: NextApiRequest, res: NextApiResponse) => {
+  res
+    .status(200)
+    .json({ name: `Hello, from ${req.url} I'm a Serverless Function'` })
+}
+```
+
+### Edge API Route
```

Comment:
```suggestion
```

Let's just show Edge API routes directly, we have other docs on normal API routes